From 61e3540d6e7b13e9a0149701183411056d1d72de Mon Sep 17 00:00:00 2001
From: Simon Xue <xxm@rock-chips.com>
Date: Wed, 16 Oct 2019 14:52:04 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808: update softreset about pcie

Delete SRST_PCIEPHY_POR, SRST_PCIEPHY_P, SRST_PCIEPHY_PIPE from PCIe
node, combo phy node already include these softreset

Change-Id: I7f1c739fbbb9b8c43ecb620b5d0655d0e1d221cb
Signed-off-by: Simon Xue <xxm@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 9a559810cc77..931f6b7e3f4a 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -336,15 +336,12 @@
 			 <&cru SRST_PCIECTL_BUTTON>, <&cru SRST_PCIECTL_PE>,
 			 <&cru SRST_PCIECTL_CORE>, <&cru SRST_PCIECTL_NSTICKY>,
 			 <&cru SRST_PCIECTL_STICKY>, <&cru SRST_PCIECTL_PWR>,
-			 <&cru SRST_PCIE_NIU_A>, <&cru SRST_PCIE_NIU_P>,
-			 <&cru SRST_PCIEPHY_POR>, <&cru SRST_PCIEPHY_P>,
-			 <&cru SRST_PCIEPHY_PIPE>;
+			 <&cru SRST_PCIE_NIU_A>, <&cru SRST_PCIE_NIU_P>;
 		reset-names = "niu-h", "niu-l", "grf-p", "ctl-p",
 			      "ctl-powerup", "ctl-mst-a", "ctl-slv-a",
 			      "ctl-dbi-a", "ctl-button", "ctl-pe",
 			      "ctl-core", "ctl-nsticky", "ctl-sticky",
-			      "ctl-pwr", "ctl-niu-a", "ctl-niu-p",
-			      "phy-por", "phy-p", "phy-pipe";
+			      "ctl-pwr", "ctl-niu-a", "ctl-niu-p";
 		rockchip,usbpciegrf = <&usb_pcie_grf>;
 		rockchip,pmugrf = <&pmugrf>;
 		status = "disabled";
-- 
2.35.3

